src/app/beer_garden/db/mongo/util.py

```diff
@@ -52,83 +52,60 @@ def ensure_roles():
     role_definition_file: Union[str, None] = config.get("auth.role_definition_file")
 
     if role_definition_file:
-        logger.info(f"Syncing role definitions from {role_definition_file}")
```

Comment:
I moved this into a separate function just to make it easier to mock in the unit tests.